src/providers/seventv/SeventvEventAPI.hpp

```diff
@@ -65,34 +62,11 @@ class SeventvEventAPI
     /** Unsubscribes from cosmetics and entitlements in a Twitch channel */
     void unsubscribeTwitchChannel(const QString &id);
 
-protected:
-    std::shared_ptr<BasicPubSubClient<seventv::eventapi::Subscription>>
-        createClient(liveupdates::WebsocketClient &client,
-                     websocketpp::connection_hdl hdl) override;
-    void onMessage(
-        websocketpp::connection_hdl hdl,
-        BasicPubSubManager<seventv::eventapi::Subscription>::WebsocketMessagePtr
-            msg) override;
+    void stop();
```

Comment:
Looks like this function is unused, and after a discussion with you, it seems like the pimpl is calling the relevant stop functions.

Removing this will break tests, so needs a little bit of finagling.